app/views/posts/show.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+    </style>
+  </head>
+
+  <body>
+   <h1> Post Info <h1/>
+   <%= @post.text %>
+    <p>Comments:
+    <% @post.query_comments.each do |comment| %>
+    <%= comment.text %>
+    </p>
+    <p>likes: 3
+    </p>
+    <% end %>
+  </body>
+</html>
```

Comment:
- [ ] I had to comment out parts of the HTML related to comments. Kindly ensure your Post show page matches the wireframe.

  Your Implementation | Wireframe
  :---:|:---:
  ![irirro](https://user-images.githubusercontent.com/36542986/225571382-fd4f39b6-772d-41d3-9ab5-6b4c05f74a92.jpg) | ![ooprr](https://user-images.githubusercontent.com/36542986/225571357-801edc33-1d76-4725-a44a-bad34aec501e.jpg)